roles/openshift_hosted/tasks/registry/secure.yml

```diff
@@ -2,21 +2,51 @@
 - name: Set fact docker_registry_route_hostname
   set_fact:
     docker_registry_route_hostname: "{{ 'docker-registry-default.' ~ (openshift_master_default_subdomain | default('router.default.svc.cluster.local', true)) }}"
+  run_once: true
+
+#- name: Create passthrough route for docker-registry
+#  oc_route:
+#    name: docker-registry
+#    namespace: "{{ openshift_hosted_registry_namespace }}"
+#    service_name: docker-registry
+#    tls_termination: passthrough
+#    host: "{{ docker_registry_route_hostname }}"
+#  run_once: true
+- name: Get the certificate contents for registry
+  local_action:
+    module: slurp
+    src: "{{ item }}"
+  register: openshift_hosted_registry_certificate_content
+  with_items:
+  - "{{ (openshift_hosted_registry_route_certificates | default({'certfile':none})).certfile }}"
+  - "{{ (openshift_hosted_registry_route_certificates | default({'keyfile':none})).keyfile }}"
+  - "{{ (openshift_hosted_registry_route_certificates | default({'cafile':none})).cafile }}"
+  when: openshift_hosted_registry_route_certificates
+
+- debug: var=openshift_hosted_registry_certificate_content
+  when: openshift_hosted_registry_route_certificates
```

Comment:
@detiber, I'll look at this.  This code was taken from another section of code for route.